### PR TITLE
C# Fundamentals : Task 4

### LINKED ISSUE #6: C# Fundamentals : Task 4

### PR DESCRIPTION
**Pull request by NaynaMallick**
closes #6

### CODE REVIEW

File: CSharpFundamentals/Set1Task4/Program.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿namespace Set1Task3
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            int a = 23;
+            int b = 7;
+            Program program = new Program();
```
Comment (ShriHumrudha) [inline CSharpFundamentals/Set1Task4/Program.cs:9]: Why do we need this?

File: CSharpFundamentals/Set1Task4/Program.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿namespace Set1Task3
```
Comment (ShriHumrudha) [inline CSharpFundamentals/Set1Task4/Program.cs:1]: Print the values of a, b, and c after every arithmetic operation. Here I see only the value of c and + *.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: CSharpFundamentals/Set1Task4/Program.cs (hunk 2 above)
Comment (NaynaMallick) [reply CSharpFundamentals/Set1Task4/Program.cs]: resolved

File: CSharpFundamentals/Set1Task4/Program.cs (hunk 1 above)
Comment (NaynaMallick) [reply CSharpFundamentals/Set1Task4/Program.cs]: resolved

File: CSharpFundamentals/Set1Task4/Program.cs [hunk 3]
```diff
@@ -8,10 +8,13 @@ public static void Main(string[] args)
             int b = 7;
             int c1 = problemAddition(a, b);
             Console.WriteLine("Value of (a++)+(--b) = " + c1);
+            Console.WriteLine();
             int c2 = problemMultiply(a, b);
             Console.WriteLine("Value of (++a)*(++b) = " + c2);
```
Comment (ShriHumrudha) [inline CSharpFundamentals/Set1Task4/Program.cs:13]: The problem states to increment a and decrement b at every step.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: CSharpFundamentals/Set1Task4/Program.cs [hunk 4]
```diff
@@ -0,0 +1,46 @@
+﻿namespace Set1Task4
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            int a = 23;
+            int b = 7;
+            int c1 = problemAddition(a, b);
```
[the remark this answers is not in the document]
Comment (NaynaMallick) [reply CSharpFundamentals/Set1Task4/Program.cs]: fixed

File: CSharpFundamentals/Set1Task4/Program.cs (hunk 3 above)
Comment (NaynaMallick) [reply CSharpFundamentals/Set1Task4/Program.cs]: fixed

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**